lib/middleware/proxy-path.js

```diff
@@ -40,6 +40,18 @@ module.exports = function (rootUrl) {
 
     return function (req, res, next) {
         var headers = {};
+        var requestOption = {
+            agent   : false,
+            hostname: parsedUrl.hostname,
+            path    : url.resolve(parsedUrl.path, pathWithIndex(req)),
+            headers : headers
+        };
+
+        // auth for private repository
+        // username:general_tokken
+        if(config.auth) {
```

Comment:
I think I also think there is no association.
I'll manage to register another issue.